lib/generate-report.js

```diff
@@ -408,6 +408,7 @@ function generateReport(options) {
         step.attachments = [];
         step.embeddings.forEach((embedding, embeddingIndex) => {
           /* istanbul ignore else */
+          embedding.data = atob(embedding.data);
```

Comment:
If we are using atob, we need to escape the Non- ASCII chracters to avoid the Invalid character error.
`embedding.data = decodeURIComponent(escape(embedding.data, 'base64'));`

But since `escape` is depericated, try `embedding.data = Buffer.from(embedding.data, 'base64')` instead

Reply:
Also, decoding an image/png will break the image in the attachment. Use `embedding.data = Buffer.from(embedding.data, 'base64')` for any non image/png media type